index.html

```diff
@@ -10,9 +10,10 @@
     while (1) {
       var result = window.prompt("日本の首都は")
```

Comment:
```suggestion
      const result = window.prompt("日本の首都は")
```
`var` は最近の javascript ではめったに使われないものになりました。
`let` または `const` を利用するようにして下さい。

Reply:
constに修正しました